src/components/Section/History/Education/Education.jsx

```diff
@@ -1,56 +1,79 @@
 import React from 'react'
-import { i18n } from '../../../../config'
-import schema from '../../../../schema'
-import validate, { EducationItemValidator } from '../../../../validators'
-import SubsectionElement from '../../SubsectionElement'
-import { Accordion } from '../../../Form'
-import { openState } from '../../../Form/Accordion/Accordion'
+import { i18n } from 'config'
+import { HISTORY, HISTORY_EDUCATION } from 'config/formSections/history'
+
+import schema from 'schema'
+import validate, { EducationItemValidator } from 'validators'
+
+import Subsection from 'components/Section/shared/Subsection'
+import { Accordion } from 'components/Form'
+import { openState } from 'components/Form/Accordion/Accordion'
+
+import connectHistorySection from '../HistoryConnector'
```

Comment:
Should we be adding relative imports? Is there a recommendation we can make like no more than one level up?

Reply:
@gidjin Yeah I am fine with that